src/validation/custom.validation.ts

```diff
@@ -279,6 +280,31 @@ export const checkBeneficialOwnerType = (beneficialOwnersStatement: string, valu
   return true;
 };
 
+export const checkBeneficialOwnerSubmission = (req) => {
```

Comment:
I'd make this method name plural - `checkBeneficialOwnersSubmission`

Reply:
Updated.